<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carInfo</class>
 <widget class="QDialog" name="carInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Подробнее о ТС</string>
  </property>
  <widget class="QLineEdit" name="grzLine">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="vinLine">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="miles">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>691</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 155, 24);
color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-radius: 4px;
border-color:rgb(41, 40, 40) ;</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Госномер</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VIN авто</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Пробег</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Проведённые ТО</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
